broker/src/tunneldigger_broker/broker.py

```diff
@@ -208,7 +208,7 @@ def close(self):
         manager instance should not be used after calling this method.
         """
 
-        for tunnel in self.tunnels.values():
+        for tunnel in list(self.tunnels.values()):
```

Comment:
the list cast looks unnecessary, because you are already calling `.values()`, which gets you `dict_values([])`, which is iterable as well.

Reply:
If we were dropping Python 2 completely, then yes, we could leave it out.
See: https://python-future.org/compatible_idioms.html#dict-keys-values-items-as-a-list
I guess this could be another step when refactoring this whole thing.

Reply:
After thinking about it a bit more: I think you are right. Since we're directly iterating over it and don't actually need it to be a list instead of a dictview I can drop the list cast.

Reply:
I actually put this back in.
If you operate directly on the values you will eventually end up with `RuntimeError: dictionary changed size during iteration` when stopping the broker. So operating on a copy with list() is helpful.